cloudsync/providers/filesystem.py

```diff
@@ -20,7 +20,7 @@
 # from watchdog.observers.polling import PollingObserver as watchdog_observer
 
 from cloudsync.event import Event
-from cloudsync.provider import Provider, Namespace
+from cloudsync.provider import Provider, NamespaceBase
```

Comment:
leave this as namespace

Reply:
This is also a consequence of the NamedTuple backwards compatibility issue.

Generally I agree with this but there is no technical reason to not instantiate this class, it should have everything one needs for a basic Namespace implementation. Could just name it something other than NamespaceBase.

Though if there is a way to have Namespace derive from NamespaceBase without breaking everything I'm all for it.